keps/sig-windows/2258-kubectl-service-log-viewer/README.md

```diff
@@ -73,37 +75,42 @@ troubleshooting.
 Provide a cluster administrator with a streaming view of logs using kubectl
 without them having to implement a client side reader or logging into the node.
 This would work for:
-- Linux worker and control plane nodes using a distribution that have systemd
-  / journald support
-- Windows worker nodes (all supported variants)
+- Services on Linux worker and control plane nodes:
+  - That have systemd / journald support.
+  - That have services that log to `/var/log/`
+- Windows worker nodes (all supported variants) that log to `C:\var\log`,
+  System and Application logs, Windows Event Logs and Event Tracing (ETW).
 
 ### Non-Goals
-Providing support for non-systemd Linux distributions.
+- Providing support for non-systemd Linux distributions.
+- Reporting logs for nodes that have config or connection issues with the cluster.
+- Getting logs from services that do not use /var/log/.
 
 ## Proposal
 
 ### Implement client for logs endpoint viewer (OS agnostic)
-Implement a client for the `/var/log/` kubelet endpoint viewer.
+- Extend `kubectl logs nodes` to work with node objects.
+- Implement a client for the `/var/log/` kubelet endpoint viewer.
 
 ```
 # See what logs are available in masters in /var/log/
-kubectl node-logs --role master --path=/
+kubectl logs nodes --role master --path=/
 
 # Display cron log file (/var/log/cron) from all masters
-kubectl node-logs --role master --path=cron
+kubectl logs nodes --role master --path=cron
 ```
 
 ### Linux distros with systemd / journald
-Suplement the the `/var/log/` endpoint viewer on the kubelet with a thin shim over
-the `journal` directory that shells out to journalctl. Then add a new option to 
-kubectl called `node-logs` that will retrieve it.
+Supplement the the `/var/log/` endpoint viewer on the kubelet with a thin shim over
+the `journal` directory that shells out to journalctl. Then extend `kubectl logs`
+to also work with node objects.
 
 ```
 # Show kubelet and crio journal logs from all masters
-kubectl node-logs node-logs --role master --path journal -u kubelet -u crio
+kubectl logs nodes --role master --path journal -s kubelet -s crio
 ```
 
-### Linux distros with systemd / journald
+### Linux distributions without systemd / journald
 Running the new "kubectl node-logs" command against nodes that do not use
 systemd / journald should return "OS not supported".
```

Comment:
this is only true for the `--path=journal` shim. Getting logs from `/var/log/` should work on all systems